commands/mining_daemon_test.go

```diff
@@ -37,3 +44,52 @@ func TestMiningGenBlock(t *testing.T) {
 
 	assert.Equal(t, sum.Add(beforeBalance, big.NewInt(1000)), afterBalance)
 }
+
+func TestMiningSealNow(t *testing.T) {
+	tf.FunctionalTest(t)
+
+	ctx, env := fastesting.NewTestEnvironment(context.Background(), t, fast.FilecoinOpts{
+		InitOpts:   []fast.ProcessInitOption{fast.POAutoSealIntervalSeconds(1)},
+		DaemonOpts: []fast.ProcessDaemonOption{fast.POBlockTime(50 * time.Millisecond)},
+	})
```

Comment:
I just landed https://github.com/filecoin-project/go-filecoin/pull/3134 you will want to add `env.RunAsyncMiner()` after this bit.